### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards> InputsWithValidation.json> Input.Toggle]: Voice access user is not able to check the checkbox of 'I accept the terms and condition' through 'Check' command.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7520]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25126.1000)
Application: Adaptive Cards UWP
Application Version: 3.0.2206.2001
Tool: Voice access

**Repro Steps:**

1. Turn on voice access.
2. Open Adaptive Cards UWP application.
3. Command: Voice access wake up'
4. Command: Show Numbers
5. Select particular number for 'InputsWithValidation.json'
6. Command: 'Check I accept the terms and conditions
7. Observe the issue.

**Actual Result:**
Voice access user is not able to check the checkbox of 'I accept the terms and condition' through 'Check I accept the terms and conditions' command. Checkbox is getting checked through command 'Check' command.

**Expected Result:**
Voice access user should be able to check the checkbox of 'I accept the terms and condition' through 'Check' command and similarly uncheck the checkbox through ''Uncheck'' command.

**Notes:**
Same issue repro with 'Red cars are better than other cars' checkbox.

**User Impact:**
People with limited mobility (cerebral palsy, quadriplegia) who use Voice Access will get impacted as user will not be able to interact with the checkbox properly if with Proper visible name checkbox is not getting checked or unchecked.

**MAS reference link:**
[[No Disruption of Accessibility Features - Liquid (microsoft.com)]](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.02.01#Zguide)

**Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://user-images.githubusercontent.com/104483005/173434513-d6d2bd94-793b-45e4-86ec-c8e92e02fa24.mp4

<img width="960" alt="MAS4 3 1 I accept the terms and condition checkbox is not getting performed" src="https://user-images.githubusercontent.com/104483005/173434654-7750f7ab-39f4-48c7-a007-58d6a09f2a1b.png">

### Card JSON

```JSON
NA
```

**Comment by Rishikesh6521** [comment]
**Fixed**

Issue is fixed, while using voice access users able to check and uncheck the 'I accept the terms and condition' and 'Red cars are better than other cars' checkbox. Please refer below attachment

https://github.com/user-attachments/assets/71dbd0af-a952-4afe-b684-5c002884df64
